src/azure-cli/azure/cli/__main__.py

```diff
@@ -31,6 +31,12 @@
 
 logger = get_logger(__name__)
 
+ALPINE_WARNING_MESSAGE = (
+    "Azure CLI 2.63 is the last version available on Alpine and will not receive updates. "
```

Comment:
`2.63.0` is more formal.

Reply:
Note that we may need a patch release related to security issues, we could use `2.63.x`

Reply:
2.63 is ok in that case.

Reply:
In the last four months, there have been CVE fixes. If we continue to fix security issues, we will release 2.63.3...

Reply:
```suggestion
    "Azure CLI 2.63.0 is the last version available on Alpine and will not receive updates. "
```